dotCMS/src/main/java/com/dotmarketing/cms/login/factories/LoginFactory.java

```diff
@@ -1,570 +1,13 @@
 package com.dotmarketing.cms.login.factories;
 
-import javax.servlet.http.Cookie;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-import javax.servlet.http.HttpSession;
-
-import com.dotcms.enterprise.BaseAuthenticator;
-import com.dotcms.enterprise.LDAPImpl;
-import com.dotcms.enterprise.PasswordFactoryProxy;
-import com.dotcms.enterprise.cas.CASAuthUtils;
-import com.dotcms.enterprise.de.qaware.heimdall.PasswordException;
-import com.dotcms.enterprise.salesforce.SalesForceUtils;
-import com.dotmarketing.business.APILocator;
-import com.dotmarketing.business.DuplicateUserException;
-import com.dotmarketing.cms.factories.PublicEncryptionFactory;
-import com.dotmarketing.cms.login.struts.LoginForm;
-import com.dotmarketing.exception.DotDataException;
-import com.dotmarketing.exception.DotSecurityException;
-import com.dotmarketing.portal.struts.DotCustomLoginPostAction;
-import com.dotmarketing.util.Config;
-import com.dotmarketing.util.Logger;
-import com.dotmarketing.util.SecurityLogger;
-import com.dotmarketing.util.UtilMethods;
-import com.dotmarketing.util.WebKeys;
-import com.liferay.portal.NoSuchUserException;
-import com.liferay.portal.auth.AuthException;
-import com.liferay.portal.auth.Authenticator;
-import com.liferay.portal.model.Company;
-import com.liferay.portal.model.User;
-import com.liferay.portal.util.PropsUtil;
-import com.liferay.util.Validator;
-
 /**
- * @author will
+ * 
+ * @deprecated use {@link com.dotcms.cms.login.LoginServicesAPI} instead.  
  *
  */
+@Deprecated
 public class LoginFactory {

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)